src/routes/sessions/[id]/+page.svelte

```diff
@@ -73,9 +73,9 @@
 
 	async function handleSessionChange() {
 		try {
-			$settingsStore.ollamaModels = (await ollamaTags()).models;
+			$settingsStore.models = await listModels();
 		} catch {
-			$settingsStore.ollamaModels = [];
+			$settingsStore.models = [];
 			toast.warning($LL.cantConnectToOllamaServer());
 		}
```

Comment:
I've been thinking about this. Wouldn't it be better to request the models in the Sessions page? Because for example if I open Hollama and go to Sessions instead of going to Settings first, then the models wouldn't be available when I click 'New session'.

Reply:
Yeah, it would be nicer to check for models when switching Sessions, but the reality is that models don't change that often and hitting the API any time `handleSessionChange` felt like we were sending way to many requests.

A more "proper solution" could be to check the models `handleSessionChange` with a debounce so we only check for models every 10 minutes or something like that. We have similar logic for [checking for version updates](https://github.com/fmaclen/hollama/blob/main/src/lib/updates.ts#L50-L54) once a week but I feel like that might add too much complexity to this PR, that's why I was suggesting keeping it only in Settings for now, and improve it later.

Reply:
I'm sorry, I didn't mean to retrieve the models when switching sessions.

As far as I understand, that would happen if we retrieve them in `src/routes/sessions/[id]/+page.svelte`. What I am saying is that we should do it in `src/routes/sessions/+page.svelte`. That way, we only retrieve the models when going to `/sessions`.

I think this is approach is better than retrieving the models in `/settings` because of the following case:
If I delete all my settings from localStorage, then `$settingsStore.models` is empty. If I enter Hollama and go to `/sessions` directly before going to `/settings`, then the models should be available so I can start a new session.

Reply:
Sorry, I missed your question. I like your suggestion of doing the check in `src/routes/sessions/+page.svelte` 👍